src/main/java/lotto/model/Number.java

```diff
@@ -0,0 +1,41 @@
+package lotto.model;
+
+import lotto.model.exception.InvalidNumberRangeException;
+import java.util.Objects;
+
+public class Number {
+
+    private final int number;
+
+    public Number(int number) {
+        if (isInvalidRange(number)) {
+            throw new InvalidNumberRangeException();
+        }
+        this.number = number;
+    }
+
+    private boolean isInvalidRange(int number) {
+        return 1 > number || number > 45;
+    }
+
+    public int getIntValue() {
+        return number;
+    }
```

Comment:
```suggestion
    public int getValue() {
        return number;
    }
```
메서드나 변수명에 자료구조를 뜻하는 네이밍은 지양해주세요! 자료구조를 변경할 때 네이밍도 변경해야합니다. 
물론 그 부분이 쉽다면 괜찮겠지만, 자료구조는 변경됐는데 변수명은 안바뀐다면... 혼란을 줄 수 있어요

Reply:
BigDecimal이나 BigInteger에서처럼 객체가 담고 있는 정보를 원하는 자료구조로 받을 수 있도록 만들고 싶었어요.
Number의 자료구조가 변경되어도 int형으로 변환해서 보낼 생각을 하고 있었고, 외부에서는 `Number의 자료구조가 머든 관심없는데 나에게 당장 필요한건 int형 값이야!`라는 의도로 사용할 수 있다고 판단되어서 메서드 명에 자료구조 명을 넣었습니다

Reply:
프로퍼티에서 갖는 `public int getValue()`에 int에 그 의미가 있지않을까요?
메서드명이 반환타입에 의존하니.. 문제가 생길여지가 있을 것 같아요.

만약 어떠한 히스토리때문에 프로퍼티는 바꿨는데, 메서드명은 바뀌지못할수도있고 `public long getIntValue()`

메서드명은 바뀌었는데, 반환타입은 못바뀔 수 있는 상황도 생길 수 있을 것 같아요. `public int getLongValue()`